backend/lib/rpc-service/service.js

```diff
@@ -0,0 +1,92 @@
+import { GLIF_RPC } from '../config.js'
+import { base64pad } from 'multiformats/bases/base64'
+import { encode as cborEncode } from '@ipld/dag-cbor'
+import { decode as jsonDecode } from '@ipld/dag-json'
+import { request } from 'undici'
+import { IpldSchema } from './ipld-schema.js'
+import { rawEventEntriesToEvent } from './utils.js'
+
+const makeRpcRequest = async (method, params) => {
+  const reqBody = JSON.stringify({ method, params, id: 1, jsonrpc: '2.0' })
+  const response = await request(GLIF_RPC, {
+    bodyTimeout: 1000 * 60,
+    headersTimeout: 1000 * 60,
+    method: 'POST',
+    headers: { 'content-type': 'application/json' },
+    body: reqBody
+  })
+  return jsonDecode(new Uint8Array(await response.body.arrayBuffer())).result
+}
+
+/*
+ A class to interact with the Lotus HTTP RPC API.
+*/
+class LotusService {
```

Comment:
- We usually use the term "client" instead of "service" for code that provides client-side helpers for accessing a remote server.
- Let's not refer to Lotus please, this should work with any JSON-RPC API provider implementing the standard methods.

Suggested name: `FilecoinRpcApiClient` or something similar (and shorter?).